src/SingleCellAnnotator.imjoy.html

```diff
@@ -941,7 +949,7 @@
               color: 'rgba(255, 255, 255, 0)'
             }),
             stroke: new Stroke({
-                color: this.current_task_name ==='segmentation'? '#f5d80b91': '#448aff53',
+                color: this.show_segmentation? '#00000000' : (this.current_task_name ==='segmentation'? '#f5d80b91': '#448aff53'),
```

Comment:
change to below?
color: this.current_task_name ==='segmentation'? (this.show_segmentation? '#f5d80b91': '#00000000'): '#448aff53' ,

Reply:
suggested changed works in my test

Reply:
I have it fixed in another PR

Reply:
```suggestion
                color: this.current_task_name ==='segmentation'? (this.show_segmentation? '#f5d80b91': '#00000000'): '#448aff53',
```